src/CMakeLists.txt

```diff
@@ -1190,8 +1190,8 @@ set(options_toml_files
 )
 string(REPLACE "toml" "cpp;" options_gen_cpp_files ${options_toml_files})
 string(REPLACE "toml" "h;"   options_gen_h_files ${options_toml_files})
-list(PREPEND options_gen_cpp_files "options/options.cpp" "options/options_public.cpp")
-list(PREPEND options_gen_h_files "options/options.h")
+list(APPEND options_gen_cpp_files "options/options.cpp" "options/options_public.cpp")
+list(APPEND options_gen_h_files "options/options.h")
```

Comment:
I don't know if the difference between `PREPEND` and `APPEND` matters here, but a few lines down, there's the use of a cvc5 CMake-based function called `list_prepend`, which implements `PREPEND` for older CMake.

Reply:
We only use the lists for fillings the dependencies for targets, so it should not matter.

Reply:
Yeah, it doesn't matter in this context but thanks for the heads-up, I didn't know/remember about `list_prepend` :)